wagtail/admin/tests/test_widgets.py

```diff
@@ -462,6 +472,38 @@ def test_render_with_free_tagging_true(self):
             ],
         )
 
+    @override_settings(TAG_SPACES_ALLOWED=True)
+    def test_tags_help_text_spaces_allowed(self):
+        """Checks that the tags help text html element content is correct when TAG_SPACES_ALLOWED is True"""
+        widget = widgets.AdminTagWidget()
+        help_text = widget.get_context(None, None, {})["widget"]["help_text"]
+
+        html = widget.render("tags", None, {})
+        help_text_html_element = self.get_help_text_html_element(html)
+
+        self.assertEqual(help_text, 'Multi-word tags with spaces will automatically be enclosed in double quotes (").')
```

Comment:
@amajai  I added this line here - please remember to ensure your test actually tests the things that are different, your test did check the help text exists but not what the text should be.